### PR TITLE
Don't blindly trim file extensions

### PR DESCRIPTION
**Pull request by nervo**
Blindly trim file extension breaks the import of libraries like fancy box, whose file name contains a dot.

    @import "fancybox/dist/scss/jquery.fancybox";

### CODE REVIEW

**Comment by gfaivre** [comment]
:+1:

**Comment by nervo** [comment]
Let me guess...
Maybe you meaned to remove *special* extensions like "scss" or "sass".

**Comment by woecifaun** [comment]
:+1:

File: sass-graph.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ var parseImports = require('./parse-imports');
 
 // resolve a sass module to a path
 function resolveSassPath(sassPath, loadPaths) {
-  // trim any file extensions
-  var sassPathName = sassPath.replace(/\.\w+$/, '');
+  // trim sass file extensions
+  var sassPathName = sassPath.replace(/\.scss$/, '');
```
Comment (xzyfer) [inline sass-graph.js]: Please update this to `/\.(css|s[ca]ss)$/`

**Comment by xzyfer** [comment]
I expect it was meant to address the sass `@imports` being extensionsless. For the moment it's safe to match against scss, sass, and css however this will not always be case since both Ruby Sass and Libsass have custom importers. 

I'll that bridge when Libsass@3.2.0 is stable. For now please update the regex.

**Comment by nervo** [comment]
Done :)

**Comment by xzyfer** [comment]
Thanks!

**Pull request merged**